nncf/experimental/torch/quantization/quantize.py

```diff
@@ -10,58 +10,54 @@
 from nncf.common.quantization.structs import QuantizationPreset
-from nncf.quantization.algorithms.post_training.algorithm import PostTrainingQuantizationParameters
+from nncf.config import NNCFConfig
+from nncf.data import Dataset
+from nncf.parameters import ModelType
+from nncf.parameters import TargetDevice
+from nncf.quantization.algorithms.bias_correction.algorithm import BiasCorrection
 from nncf.quantization.algorithms.post_training.algorithm import PostTrainingQuantization
-from nncf.quantization.algorithms.min_max.algorithm import MinMaxQuantization
+from nncf.quantization.algorithms.post_training.algorithm import PostTrainingQuantizationParameters
+from nncf.scopes import IgnoredScope
 from nncf.torch.model_creation import create_nncf_network
 
 
 def quantize_impl(
-        model: torch.nn.Module,
-        calibration_dataset: Dataset,
-        preset: QuantizationPreset,
-        target_device: TargetDevice,
-        subset_size: int,
-        fast_bias_correction: bool,
-        model_type: Optional[ModelType] = None,
-        ignored_scope: Optional[IgnoredScope] = None) -> torch.nn.Module:
+    model: torch.nn.Module,
+    calibration_dataset: Dataset,
+    preset: QuantizationPreset,
+    target_device: TargetDevice,
+    subset_size: int,
+    fast_bias_correction: bool,
+    model_type: Optional[ModelType] = None,
+    ignored_scope: Optional[IgnoredScope] = None,
+) -> torch.nn.Module:
     """
     Experimental implementation of the `quantize()` method for the PyTorch backend.
     """
-    if fast_bias_correction is False:
-        raise ValueError(f'fast_bias_correction={fast_bias_correction} is not '
-                          'supported')
-    nncf_logger.warning('Bias correction and fast bias correction algorithms'
-                        ' are not supported by Torch backend by now.')
 
     dataset_iter = iter(calibration_dataset.get_inference_data())
     input_shape = tuple(next(dataset_iter).shape)
-    nncf_config = NNCFConfig({
-        'input_info': {
-            'sample_size': input_shape
-        }
-    })
+    nncf_config = NNCFConfig({'input_info': {'sample_size': input_shape}})
     model.eval()
     nncf_network = create_nncf_network(model, nncf_config)
-    params = PostTrainingQuantizationParameters(number_samples=subset_size,
-                                                preset=preset,
-                                                target_device=target_device,
-                                                ignored_scopes=ignored_scope,
-                                                model_type=model_type)
+    params = PostTrainingQuantizationParameters(
+        number_samples=subset_size,
+        preset=preset,
+        target_device=target_device,
+        ignored_scopes=ignored_scope,
+        model_type=model_type,
+        fast_bias_correction=fast_bias_correction,
+    )
+
+    nncf_logger.warning('Bias correction algorithms are not supported by Torch backend by now.')
```

Comment:
There is removed all algorithms expect MinMax (BiasCorrection and FastBiasCorrection). I changed to remove only BiasCorrection.
I think BiasCorrection in warning it's correct.

Reply:
The warning is not correct. 
`Bias correction algorithms` = `FastBiasCorrection` + `BiasCorrection`. 
Since `FastBiasCorrection` is applicable for PyTorch, the warning is not correct.

Reply:
Is it means that i should return
```
min_max_params = params.algorithms[MinMaxQuantization]
params.algorithms = {MinMaxQuantization: min_max_params}
```